### ISSUE TITLE
R6 Layout 2 question

### ISSUE THREAD

**Issue by jsodeman** [dvdizon/kbd75-qmk-guide#2]
When picking the KBD75 configuration in QMK builder the results have the Fn button on the bottom row between RALT and RCTRL which is different that my configuration. I just want to change Pause to be Delete as shown in the layouts on KBDfans, the instructions are clear on how to do that. What isn't clear is if I have to remap the Fn key to be to the left of Pause like it currently is, or if it'll stay the same when I apply the Pause->Del change.

In other words does the update only affect the keys that I've changed in the Builder (Pause), or do all the other keys get set to what's shown in the Builder too (Pause, Slck etc)?

**Comment by dvdizon** [comment]
The firmware flash overrides the whole layout, what you see on the Builder is what you get, so it's a good idea to create a copy of your old layout, and then make the new keybinding.
